src/status_im/ui/screens/privacy_and_security_settings/views.cljs

```diff
@@ -61,9 +66,14 @@
                       :on-press                #(re-frame/dispatch
                                                  [:multiaccounts.ui/preview-privacy-mode-switched
                                                   ((complement boolean) preview-privacy?)])}]
-
-      (comment
-        {:container-margin-top 8
-         :size                 :small
-         :title                :t/delete-my-account
-         :theme                :negative})]]))
+      ;; TODO(rasom): remove this condition when kk support will be added
+      (when-not keycard?
+        [separator])
+      (when-not keycard?
+        [quo/list-item
+         {:size                :small
+          :theme               :negative
+          :title               (i18n/label :t/delete-my-profile)
+          :on-press            #(re-frame/dispatch [:navigate-to ::events/delete-profile])
```

Comment:
Seems like here should be a screen name instead of event?

Reply:
fixed